src/AccessibilityInsights.CommonUxComponents/Controls/DualFabricIconControl.xaml.cs

```diff
@@ -0,0 +1,204 @@
+
+            set { SetValue(ForegroundProperty, value); }
+        }
+
+        public static void OnForegroundChanged(DependencyObject o, DependencyPropertyChangedEventArgs e)
+        {
+            DualFabricIconControl sender = o as DualFabricIconControl;
+
+            if (sender != null)
+
+            {
+                sender.fabicnFront.Foreground = sender.Foreground;
+            }
+        }
+
+        #endregion
+
+        /// <summary>
+        /// Name of icon in back to be drawn
+        /// </summary>
+        #region GlyphNameBack (Dependency Property)
+
+        public static readonly DependencyProperty GlyphNameBackProperty =
+            DependencyProperty.Register("GlyphNameBack", typeof(FabricIcon?), typeof(DualFabricIconControl), new PropertyMetadata(default(FabricIcon?), OnGlyphNameBackChanged));
+
+        public FabricIcon? GlyphNameBack
+        {
+            get { return (FabricIcon)GetValue(GlyphNameBackProperty); }
+
+            set { SetValue(GlyphNameBackProperty, value); }
+        }
+
+        public static void OnGlyphNameBackChanged(DependencyObject o, DependencyPropertyChangedEventArgs e)
+        {
+            DualFabricIconControl sender = o as DualFabricIconControl;
+
+            if (sender != null)
+            {
+                sender.fabicnBack.GlyphName = sender.GlyphNameBack;
+            }
+        }
+
+        #endregion
+
+        /// <summary>
+        /// Size of icon in back
+        /// </summary>
+        #region IconSizeBack (Dependency Property)
+
+        public static readonly DependencyProperty IconSizeBackProperty =
+            DependencyProperty.Register("IconSizeBack", typeof(double), typeof(DualFabricIconControl), new PropertyMetadata(1.0, OnIconSizeBackChanged));
+
+        public double IconSizeBack
+        {
+            get { return (double)GetValue(IconSizeBackProperty); }
+
+            set { SetValue(IconSizeBackProperty, value); }
+        }
+
+        public static void OnIconSizeBackChanged(DependencyObject o, DependencyPropertyChangedEventArgs e)
+        {
+            DualFabricIconControl sender = o as DualFabricIconControl;
+
+            if (sender != null)
+            {
+
+                sender.fabicnBack.FontSize = sender.IconSizeBack;
+
+            }
+        }
+
+        #endregion
+
+        /// <summary>
+        /// Color of icon in back
+        /// </summary>
+        #region Background (Dependency Property)
+
+        new public static readonly DependencyProperty BackgroundProperty =
+            DependencyProperty.Register("Background", typeof(Brush), typeof(DualFabricIconControl), new PropertyMetadata(default(Brush), OnBackgroundChanged));
+
+        new public Brush Background
+        {
+            get { return (Brush)GetValue(BackgroundProperty); }
+
+            set { SetValue(BackgroundProperty, value); }
+        }
+
+        public static void OnBackgroundChanged(DependencyObject o, DependencyPropertyChangedEventArgs e)
+
+        {
```

Comment:
Extra newlines are making this code look odd...

Reply:
Agreed